### PR TITLE
feat(cli): add --parameter flag to exp scaletest command

### PR DESCRIPTION
**Pull request by johnstcn**
This PR adds support for passing parameters to workspaces built by `exp scaletest`.

### CODE REVIEW

File: cli/exp_scaletest.go [hunk 1]
```diff
@@ -597,11 +599,29 @@ func (r *RootCmd) scaletestCreateWorkspaces() *clibase.Cmd {
...
 
+			cliRichParameters, err := asWorkspaceBuildParameters(parameterFlags.richParameters)
+			if err != nil {
+				return xerrors.Errorf("can't parse given parameter values: %w", err)
+			}
+
+			richParameters, err := prepWorkspaceBuild(inv, client, prepWorkspaceBuildArgs{
+				Action:           WorkspaceCreate,
+				Template:         tpl,
+				NewWorkspaceName: "scaletest-%", // TODO: the scaletest runner will pass in a different name here. Does this matter?
```
Comment (johnstcn) [inline cli/exp_scaletest.go:610]: @mtojek Does the value here only matter for showing the plan output?

Comment (mtojek) [reply cli/exp_scaletest.go:610]: Yes, only for dry-run purposes. Not sure what happens if there is a workspace with such name exists. In the worst case, you can append a random number.

File: cli/exp_scaletest.go [hunk 2]
```diff
@@ -525,6 +525,8 @@ func (r *RootCmd) scaletestCreateWorkspaces() *clibase.Cmd {
 
 		useHostUser bool
 
+		parameterFlags workspaceParameterFlags
```
Comment (mtojek) [inline cli/exp_scaletest.go:528]: FYI This may also pull `--build-options`, not sure if you want to support them too. Probably not.

**Review by mtojek — APPROVED** [review]
ok!

File: cli/exp_scaletest.go (hunk 2 above)
Comment (johnstcn) [reply cli/exp_scaletest.go:528]: I don't see a need at present. We don't rebuild workspaces in this command, only create.

**Pull request merged**